public/css/style.css

```diff
@@ -0,0 +1,21 @@
+.form-group input[type="checkbox"] {
```

Comment:
Css seems pretty good. But when writing it targe class rather than checkbox or any nodes/form element. So I would create a class in css called `.completed` or whatever makes sense. 

Future study assets built pipeline. (Something we need to have a look at). But Good stuff with CSS.